build/src/components/05-gallery/gallery-fs-mobile-scroll.js

```diff
@@ -0,0 +1,76 @@
+import enquire from 'enquire.js';
+import Gallery from './gallery';
+
+/**
+ * A variant with vertical scrolling on mobile phones.
+ */
+class GalleryFsMobileScroll extends Gallery {
+
+  /**
+   * @inheritdoc
+   */
+  static get type() {
+    return 'fs-mobile-scroll';
+  }
+
+  /**
+   * Gets the swiper config for the mobile variant.
+   *
+   * Overrides defaultConfig (see base class).
+   *
+   * @returns {Object}
+   */
+  get mobileConfig() {
+    let self = this;
+    return {
+      freeMode: true,
+      direction: 'vertical',
+      spaceBetween: 10,
+      autoHeight: true,
+      slidesPerView: 'auto',
+      watchSlidesVisibility: true,
+      centeredSlides: false,
+      loop: false,
+      preloadImages: false,
+      lazy: true,
+      pagination: {
+        el: '.gallery__pagination',
+        type: 'custom',
+        renderCustom: function (swiper, current, total) {
+          // The total provided by swiper will be the amount of loaded slides.
+          // We provide the real total value by counting the slides.
+          let slides = self.content.querySelectorAll('.swiper-wrapper')[0].querySelectorAll('.gallery-slide');
```

Comment:
This can be simplified to:
`self.content.querySelector('.swiper-wrapper').querySelectorAll('.gallery-slide');`

Also the comment should state why the total provided by swiper may not be the real amount of slides